capsules/classifier_face_emotion_openvino/config.py

```diff
@@ -0,0 +1 @@
+emotion_types = ['neutral', 'happy', 'sad', 'surprise', 'anger']
```

Comment:
Tbh having a config for one line probably isn't necessary :P You can have it in the backend and import it from capsule.py to do the same thing